ios/Nebra/Info.plist

```diff
@@ -54,6 +56,8 @@
 	<string>Allow Nebra Hotspot to connect to and manage your Hotspots</string>
 	<key>NSBluetoothPeripheralUsageDescription</key>
 	<string>Allow Nebra Hotspot to connect to and manage your Hotspots</string>
+	<key>NSCameraUsageDescription</key>
```

Comment:
We don't actually use camera/qr. Can't this be removed? Similar to https://github.com/NebraLtd/maker-starter-app/pull/66

Reply:
"ITMS-90683: Missing Purpose String in Info.plist - Your app's code references one or more APIs that access sensitive user data. The app's Info.plist file should contain a NSCameraUsageDescription key with a user-facing purpose string explaining clearly and completely why your app needs the data. Starting Spring 2019, all apps submitted to the App Store that access user data are required to include a purpose string. If you're using external libraries or SDKs, they may reference APIs that require a purpose string. While your app might not use these APIs, a purpose string is still required. You can contact the developer of the library or SDK and request they release a version of their code that doesn't contain the APIs. Learn more (https://developer.apple.com/documentation/uikit/core_app/protecting_the_user_s_privacy)"

Above is a message from App store indicating why our deployment to testflight was flagged missing compliance. We have a library installed "expo-camera" which is a camera API. We have only commented out the link to the QR code scanning. We still have the codes for the QR code scanning. So i would suggest we keep <key>NSCameraUsageDescription</key> for now and probably take it off if we decide to totally do away with the QR code scanning.